### PR TITLE
v1.0.25 - Kodi 19 / Py3 compatibility and some improvements

### PR DESCRIPTION
**Pull request by AnonTester**

### CODE REVIEW

**Comment by gade01** [comment]
I just tested this with latest nightly, and everything works as expected.

**Comment by gade01** [comment]
Remember to bump 
`<import addon="xbmc.python" version="2.13.0"/>`
to v3.0.0

File: addon.xml [hunk 1]
```diff
@@ -1,20 +1,21 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-	<addon id="script.skin.helper.skinbackup" name="Skin Helper Service Skin Backup" version="1.0.19" provider-name="marcelveldt">
-	<requires>
-		<import addon="xbmc.python" version="2.13.0"/>
-		<import addon="xbmc.addon" version="12.0.0"/>
-	</requires>
+    <addon id="script.skin.helper.skinbackup" name="Skin Helper Service Skin Backup" version="1.0.20" provider-name="marcelveldt, wuff">
+    <requires>
+        <import addon="xbmc.python" version="2.13.0"/>
```
Comment (gade01) [inline addon.xml]: Bump to v3.0.0

**Comment by AnonTester** [comment]
Summary of last updates and changes:

v1.0.25 (2020-10-04)
- remove duplicate slash in zip path preventing zip to be copied in backup
- sort backups in descending order and only show backupdate

v1.0.24 (2020-09-15)
- replace ListItem iconImage with setArt

v1.0.23 (2020-09-12)
- Fix icon

v1.0.22 (2020-09-07)
- Replace xbmc.translatePath wiht xbmcvfs.translatePath for Kodi 19 compat

**Comment by Ch1llb0** [comment]
> I just tested this with latest nightly, and everything works as expected.

What's the holdup here?

Would be great to be able to use this script on v19 builds again with my skin. Atm, I have to comment out sections that need this script as it can't be installed automatically from the Kodi repo. 👍🏻

**Comment by AnonTester** [comment]
> What's the holdup here?

Original developer is not involved with Kodi any longer due to lack of time. No idea who or if someone else has write access to this github.

> Would be great to be able to use this script on v19 builds again with my skin. Atm, I have to comment out sections that need this script as it can't be installed automatically from the Kodi repo. 👍🏻

This as well as other addons I've adjusted for Kodi19 and/or fixed are on my github and included in my kodi repository for you to use. Add the repo to your kodi for auto-updates or download and install the relevant addon zip from there manually: https://github.com/AnonTester/kodi-repo

**Comment by Ch1llb0** [comment]
Does @gade01 maybe know who has control over this Git repo and who might consequently be able to merge this once done?

**Comment by gade01** [comment]
@Ch1llb0 @AnonTester I can merge, but I don't know much about python. So I'm really able to review.

**Comment by janinop** [comment]
I couldn't find a way to download it. I have Kodi 19, Arctic zephyr 2 and I want to back up..but it doesn't work.

**Pull request merged**